api/loki.ts

```diff
@@ -13,7 +13,7 @@ import { EnvConfigService } from "../src/services/EnvConfigService";
 export default async (req: NowRequest, res: NowResponse): Promise<void> => {
   const groupMeInfo = req.body as GroupMeInfo;
 
-  container.register(EnvConfigService, { useValue: new EnvConfigService({ groupID: groupMeInfo.group_id }) });
+  container.register(EnvConfigService, { useValue: new EnvConfigService() });
```

Comment:
I think this line can actually be removed since it is being injected somewhere...Needs to be tested though